projects/demo/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@polkadot/api": "^7.13.1",
-    "@substrate/connect": "^0.6.5",
```

Comment:
FTR this will no longer be a dependency once #909 is merged, so this PR in its current shape will have conflicts after #909 is merged